connectors/byoei.py

```diff
@@ -63,13 +65,14 @@ class Bulker:
     This class runs a coroutine that gets operations out of a `queue` and collects them to
     build and send bulk requests using a `client`
 
-    The bulk requests are controlled in several ways:
+    Arguments:
+
+    - `client` -- an `connectors.es.ESClient` instance
```

Comment:
```suggestion
    - `client` -- a `connectors.es.ESClient` instance
```

Reply:
What about "an instance of `TYPE`" convention for such cases?

Reply:
I like that more ☝🏻

Reply:
Updated my comment to reflect that